### PR TITLE
Added Logic for Rock Paper Scissors.

### PR DESCRIPTION
**Pull request by AngryGrizzlyBear**
[Step 5: Write the logic to play a single round](https://www.theodinproject.com/lessons/foundations-rock-paper-scissors#step-5-write-the-logic-to-play-a-single-round)
Your game will be played round by round. You will write a function that takes the human and computer player choices as arguments, plays a single round, increments the round winner’s score and logs a winner announcement.

- [x] 1. Create a new function named playRound.
- [ ] 2. Define two parameters for playRound: humanChoice and computerChoice. Use these two parameters to take the human and computer choices as arguments.
- [x] 3. Make your function’s humanChoice parameter case-insensitive so that players can input “rock”, “ROCK”, “RocK”, or other variations.
- [x] 4. Write the code for your playRound function to console.log a string value representing the round winner, such as: “You lose! Paper beats Rock”.
- [x] 5. Increment the humanScore or computerScore variable based on the round winner.

Example code:
Screen Shot 2024-05-28 at 12 54 17 PM

Problem:
My code only returns the else statement in my if else statement.

Steps to attempt to solve:
1. I rewrote my code a few times to see if there were any errors in my operation.
2. I corrected an error I noted in the example code. I learned it wasn't calling or using any of the functions as arguments.
3. I looked at some example code for this to see the issue. The good news is that this is a common error, but the bad news is that the people who answered the question didn't make much sense for a beginner. I did not copy the code and opted to try to understand the logic behind writing their code. Unfortunately, in most cases, this question is solved in a complex way that deviates slightly from the assignment. There's still one more step to go, so I'm assuming it's better to understand how and why something works over copying and pasting.

### CODE REVIEW

File: javascript/rock-paper-scissors.js [hunk 1]
```diff
@@ -1,35 +1,58 @@
...
 // Test that your function returns what you expect using console.log or the browser developer tools before advancing to the next step.
-let humanScore = 0;
-let computerScore = 0;
-
 function getComputerChoice() {
-    num = Math.floor(Math.random()*3);
+    num = Math.floor(Math.random() * 3);
 
-    if (num == 0){
+    if (num == 0) {
```
Comment (cbrandao18) [inline javascript/rock-paper-scissors.js:15]: commetn 1

File: javascript/rock-paper-scissors.js [hunk 2]
```diff
@@ -1,35 +1,58 @@
...
         console.log("rock")
     } else if (num == 1) {
         console.log("paper")
     } else if (num == 2) {
         console.log("scissors")
     } else {
-        console.log ("try again!")
+        console.log("try again!")
```
Comment (cbrandao18) [inline javascript/rock-paper-scissors.js:22]: and here

File: javascript/rock-paper-scissors.js [hunk 3]
```diff
@@ -1,35 +1,58 @@
...
     let choice = prompt("Pick Rock, Paper, or Scissors")
-    console.log(choice.toLowerCase());    
+    console.log(choice.toLowerCase());
 }
-// Create two new variables named humanScore and computerScore in the global scope.
-// Initialize those variables with the value of 0.
 
+// Create a new function named playRound.
+// Define two parameters for playRound: humanChoice and computerChoice. Use these two parameters to take the human and computer choices as arguments.
+// Make your function’s humanChoice parameter case-insensitive so that players can input “rock”, “ROCK”, “RocK”, or other variations.
```
Comment (cbrandao18) [inline javascript/rock-paper-scissors.js:37]: make tehis better

**Review by cbrandao18 — APPROVED** [review]

File: javascript/rock-paper-scissors.js (hunk 2 above)
Comment (cbrandao18) [inline javascript/rock-paper-scissors.js]: you should return a value in this function

File: javascript/rock-paper-scissors.js [hunk 4]
```diff
@@ -1,35 +1,58 @@
...
 // Create a new function named getHumanChoice.
 // Write the code so that getHumanChoice will return one of the valid choices depending on what the user inputs.
 // Test what your function returns by using console.log.
 
-function getHumanChoice(){
+function getHumanChoice() {
     let choice = prompt("Pick Rock, Paper, or Scissors")
-    console.log(choice.toLowerCase());    
+    console.log(choice.toLowerCase());
 }
```
Comment (cbrandao18) [inline javascript/rock-paper-scissors.js]: also here - should return the choice

**Review by cbrandao18 — CHANGES_REQUESTED** [review]

File: javascript/rock-paper-scissors.js (hunk 2 above)
Comment (AngryGrizzlyBear) [reply javascript/rock-paper-scissors.js]: Resolved!

File: javascript/rock-paper-scissors.js (hunk 4 above)
Comment (AngryGrizzlyBear) [reply javascript/rock-paper-scissors.js]: Resolved!

**Pull request merged**